### PR TITLE
changed according to new main branch changes footer.jsx

### PR DESCRIPTION
**Pull request by devxMani**
#56 is done you can check 
working fine for me 
image

### CODE REVIEW

File: frontend/src/Components/Footer.jsx [hunk 1]
```diff
@@ -57,7 +58,6 @@ const Footer = () => {
     };
 
     try {
-      //This is the URL you can use if you are working on local machine if creating error in finding backendURL so use this in fetch request = 'http://localhost:8080/api/feedback
       const response = await fetch(`${backendURL}/api/feedback`, {
```
Comment (AlfiyaSiddique) [inline frontend/src/Components/Footer.jsx:61]: @devxMani why remove this comment?
You have removed all the comments wherever it wsa before. Please put those comments back

Comment (devxMani) [reply frontend/src/Components/Footer.jsx:61]: @AlfiyaSiddique i'll create a new pr for this

**Comment by devxMani** [comment]
@AlfiyaSiddique if this working fine can you add level 2 to this

**Comment by AlfiyaSiddique** [comment]
@devxMani any updates?

**Comment by devxMani** [comment]
@AlfiyaSiddique I'll do it give me some time

**Comment by AlfiyaSiddique** [comment]
@devxMani any updates? You can give us a date by when you will continue, since there are chances of PR getting closed due to no activity

**Comment by AlfiyaSiddique** [comment]
@devxMani this Pr will get closed tomorrow if no activity found

**Comment by devxMani** [comment]
@AlfiyaSiddique close..

**Pull request closed without merging**